perfkitbenchmarker/benchmark_spec.py

```diff
@@ -517,6 +525,13 @@ def ConstructSparkService(self):
                           'service'.format(name, spark_service.PKB_MANAGED))
         self.vms_to_boot[name] = spec
 
+  def ConstructVPNService(self):
+    """Create the VPNService object."""
+    if self.config.vpn_service is None:
+      return
+    vpn_service_spec = self.config.vpn_service
+    self.vpn_service = vpn_service.VPNService(vpn_service_spec)
```

Comment:
I would just make this one line:
  self.vpn_service = vpn_service.VPNService(self.config.vpn_service)

Reply:
Fixed.